### PR TITLE
Update console.mdx

### PR DESCRIPTION
**Pull request by jdgemm**
updates for api docs

### CODE REVIEW

File: docs/api/console.mdx [hunk 1]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+
+_Body Parameters_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| name (required)    | _string_ | A human-friendly name for the device |
+| app_eui (required) | _string_ | LoRaWAN Application EUI              |
+| app_key (required) | _string_ | LoRaWAN Application Key              |
+| dev_eui (required) | _string_ | LoRaWAN Device EUI                   |
+| Config_profile_id (required) | _string_ | Config profile uuid to attach                   |
```
Comment (kent-williams) [inline docs/api/console.mdx]: Does this parameter actually start with a capital C? This would diverge from the norm.

Comment (jdgemm) [reply docs/api/console.mdx]: updated case

**Review by kent-williams — DISMISSED** [review]

File: docs/api/console.mdx [hunk 2]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+| key (required)    | _string_ | your API key |
+| content-type (required) | _string_ | application/json              |
+
+_Body Parameters_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| token (required)    | _string_ | Your token, authorization key, uplink secret depending on pre-built integration.|
+| name (required) | _string_ | A unique name for the Integration           |
+| type (required) | _string_ | “Akenza” “ubidots” “tago” “datacake”             |
```
Comment (vicmgs) [inline docs/api/console.mdx]: uncap Akenza

File: docs/api/console.mdx [hunk 3]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| key (required)    | _string_ | your API key |
+
+_Path Parameters_
+
+| Parameter Name | Type     | Description                      |
+| -------------- | -------- | -------------------------------- |
+| Integration_id (required)  | _string_ | UUID for removed integration |
```
Comment (vicmgs) [inline docs/api/console.mdx]: uncap Integration_id

File: docs/api/console.mdx [hunk 4]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+    },
+    Integration {
+        id: "d20d5d32-f699-40a7-a5a7-b9a49dfe2b4a",
+        name: "Tommy Douglas",
+        labels: [],
+        devices: [],
+    },
+```
+
+404: Not Found
```
Comment (vicmgs) [inline docs/api/console.mdx]: remove this section

File: docs/api/console.mdx [hunk 5]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| key (required)    | _string_ | your API key |
+
+_Path Parameters_
+
+| Parameter Name | Type     | Description                      |
+| -------------- | -------- | -------------------------------- |
+| Integration_id (required)  | _string_ | UUID of integration |
```
Comment (vicmgs) [inline docs/api/console.mdx]: uncap Integration

File: docs/api/console.mdx [hunk 6]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+```
+GET https://staging-console.helium.wtf/api/v1/integrations?name=
+```
+
+<Tabs
+  block={true}
+  defaultValue="request"
+  values={[{"label":"Request","value":"request"},{"label":"Response","value":"response"}]}>
+<TabItem value="request">
+
```
Comment (vicmgs) [inline docs/api/console.mdx:933]: missing key header

File: docs/api/console.mdx [hunk 7]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+</TabItem>
+</Tabs>
+
+---
+
+### Search for an Integration by Name
+
+```
+GET https://staging-console.helium.wtf/api/v1/integrations?name=
```
Comment (vicmgs) [inline docs/api/console.mdx]: ?name=your_integration_name

File: docs/api/console.mdx [hunk 8]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+
+</TabItem>
+</Tabs>
+
+---
+
+### View all flows for integration
+
+```
+GET https://staging-console.helium.wtf/api/v1/flows?integration_id=xxx
```
Comment (vicmgs) [inline docs/api/console.mdx]: ?integration_id=your_integration_id

File: docs/api/console.mdx [hunk 9]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| key (required)    | _string_ | your API key |
+
+_Query Parameters_
+
+| Parameter Name | Type     | Description                      |
+| -------------- | -------- | -------------------------------- |
+| Integration_id (required)  | _string_ | UUID of integration |
```
Comment (vicmgs) [inline docs/api/console.mdx]: uncap Integration_id

File: docs/api/console.mdx [hunk 10]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+| key (required)    | _string_ | your API key |
+| content-type (required) | _string_ | application/json              |
+
+_Body Parameters_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| integration_id (required)    | _string_ | UUID of integration|
+| label_id / device_id (required) | _string_ | UUID           |
+| Function_id (optional) | _string_ | UUID             |
```
Comment (vicmgs) [inline docs/api/console.mdx]: uncap Function_id

File: docs/api/console.mdx [hunk 11]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+
+</TabItem>
+<TabItem value="response">
+
+201: Created
```
Comment (vicmgs) [inline docs/api/console.mdx]: 200: Ok

Flow with integration created successfully

File: docs/api/console.mdx [hunk 12]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+
+_Body Parameters_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| name (required)    | _string_ | A human-friendly name for the device |
+| app_eui (required) | _string_ | LoRaWAN Application EUI              |
+| app_key (required) | _string_ | LoRaWAN Application Key              |
+| dev_eui (required) | _string_ | LoRaWAN Device EUI                   |
+| config_profile_id (required) | _string_ | Config profile uuid to attach                   |
```
Comment (vicmgs) [inline docs/api/console.mdx]: optional

File: docs/api/console.mdx [hunk 13]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+| label_ids | Array of strings | Array of label uuids to attach to device               |
```
Comment (vicmgs) [inline docs/api/console.mdx]: optional

File: docs/api/console.mdx [hunk 14]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+| ------------------ | -------- | ------------------------------------ |
+| key (required)    | _string_ | your API key |
+| content-type (required) | _string_ | application/json              |
+
+_Body Parameters_
+
+| Parameter Name     | Type     | Description                          |
+| ------------------ | -------- | ------------------------------------ |
+| name (required)    | _string_ | A human-friendly name for the device |
+| config_profile_id (required) | _string_ | Config profile uuid to attach                   |
```
Comment (vicmgs) [inline docs/api/console.mdx]: optional

File: docs/api/console.mdx [hunk 15]
```diff
@@ -686,3 +686,695 @@ Device removed from label successfully
...
+200: OK
+
+</TabItem>
+</Tabs>
+
+---
+
+### Update Devices Active Status (New)
+```
+PUT https://staging-console.helium.wtf/api/v1/devices/active?
```
Comment (vicmgs) [inline docs/api/console.mdx]: active?dev_eui=your_dev_eui

**Review by kent-williams — APPROVED** [review]

**Pull request merged**